change-cuda-versions.sh

```diff
@@ -1,26 +1,28 @@
 #!/usr/bin/env bash
 
-################################################################################
-# Copyright (c) 2015-2018 Skymind, Inc.
 #
-# This program and the accompanying materials are made available under the
-# terms of the Apache License, Version 2.0 which is available at
-# https://www.apache.org/licenses/LICENSE-2.0.
+# /* ******************************************************************************
+#  *
```

Comment:
"Copyright (c) 2021 Deeplearning4j Contributors" missing here and a few other places?

Reply:
@saudet  I moved that to a NOTICES.txt instead. See: https://github.com/eclipse/deeplearning4j/blob/ag_dev_updates_KonduitAI_master/NOTICE.txt

Reply:
I know, but it should say at least something like "Copyright (c) 2021 Deeplearning4j Contributors" and/or "See the NOTICE file distributed with this work for additional information regarding copyright ownership." I don't think we can leave it completely devoid of copyright. Did you confirm with Eclipse PMC?

Reply:
@saudet  following apache arrow here (I assume it's the same standard for all ASF project that uses notices files):
 Licensed to the Apache Software Foundation (ASF) under one or more contributor
  license agreements. **See the NOTICE file distributed with this work for additional
  information regarding copyright ownership.** The ASF licenses this file to
  You under the Apache License, Version 2.0 (the "License"); you may not use
  this file except in compliance with the License. You may obtain a copy of
  the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required
  by applicable law or agreed to in writing, software distributed under the
  License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
  OF ANY KIND, either express or implied. See the License for the specific
  language governing permissions and limitations under the License.
  
  How does thi slook?

Reply:
Yes, I think that would be fine.

Reply:
@saudet https://github.com/eclipse/deeplearning4j/pull/9170/commits/c715aea405980eb043420af86c671032bbb78ec6

Reply:
Looks acceptable, but all files other than the *.java ones, mainly *.cpp, *.h, but others too, need to be updated as well.

Reply:
Hmm..I wonder what happened with intellij...let me check the latest commit and see.

Reply:
@saudet  I got what I could find. Let me scan for other ones. It turns out intellij's automatic refactoring doesn't catch everything. I just did a find and replace all on what I could find.

Reply:
There's still a bunch of them missing:
```
$ grep -r "See the NOTICE file distributed with this work for additional"  | wc -l
7190
$ grep -r "Unless required by applicable law or agreed to in writing, software" | wc -l
8303
```

Reply:
@saudet  this should be done. I manually scanned everything and just moved the attic out for now.

Reply:
Still seeing missing updates:
```
$ grep -r "See the NOTICE file distributed with this work for additional" | cut -f 1 -d: | sort -u | wc -l
7330
$ grep -r "Unless required by applicable law or agreed to in writing, software" | cut -f 1 -d: | sort -u | wc -l
7540
```

Reply:
@saudet ran this test and deleted the nlp modules, the only thing I'm not sure what to do with is the 3rd party src/main/protobuf/tf under nd4j-api. I'm tempted to leave those as is. Mind looking again? Thanks for being so helpful with this.

Reply:
Looks OK! Since you're not removing the "Copyright" bit from those files from TF, that should be alright yes.